src/app/containers/Include/canonical/Idt2/index.jsx

```diff
@@ -19,19 +19,25 @@ const Include = styled.div`
 
 const DISALLOWED_SCRIPTS = /js\/verticalChart|horizontalChart|lineChart|pieChart|simpleMap|table\./;
 
-const Idt2Canonical = ({ html, imageBlock }) => {
+const Idt2Canonical = ({ html, imageBlock, indexOfInclude }) => {
```

Comment:
We could call this `index` - `indexOfInclude` suggests to me there might be another `indexOfSomething` I should be concerned about.

```jsx
<CanonicalIncludeContainer html={} type="" indexOfInclude={0} />
``` 
```jsx
<CanonicalIncludeContainer html={} type="" index={0} />
```

`indexOfType` was use in the Social Embed implementation, because Embeds were indexed on type - Twitter, Instagram, YouTube.